index.html

```diff
@@ -1,108 +1,127 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
+  <head>
+    <meta charset="UTF-8" />
     <title>Fork Me! FCC: Test Suite Template</title>
-    <script src="https://kit.fontawesome.com/8ea4cd9e84.js" crossorigin="anonymous"></script>
-    <link rel="stylesheet" href="css/style.css">
-</head>
+    <script
+      src="https://kit.fontawesome.com/8ea4cd9e84.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
 
-<body>
+  <body>
     <header id="header">
-        <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="original trombones logo">
-        <nav id="nav-bar">
-            <div class="nav-link">
-                <a href="#Features">Features</a>
-                <a href="#How It Works">How It Works</a>
-                <a href="#Pricing">Pricing</a>
-            </div>
-        </nav>
+      <img
+        id="header-img"
+        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+        alt="original trombones logo"
+      />
+      <nav id="nav-bar">
```

Comment:
Please try and avoid using too many id selectors in your code. This is because if another element in the page uses the same/similar style, you would have to write the same CSS again.